docs/intro.qmd

```diff
@@ -5,25 +5,24 @@ sidebar_position: 1
 slug: /
 ---
 
-# Introduction
-
-This document is an attempt to do some typical table data manipulation
-using [PRQL](https://prql-lang.org/) and [DuckDB](https://duckdb.org/).
-
-PRQL is a in development modern language for data manipulation, and can be compiled to SQL.
-DuckDB is a modern in-memory SQL OLAP ([very fast](https://duckdblabs.github.io/db-benchmark/))
-database management system.
-
 ## Motivation
 
-CLI, JavaScript, Python, R, etc., we can combine PRQL compiler and DuckDB in various places
-to manipulate data with PRQL.
+I wanted to compare PRQL and the latest DuckDB SQL
+(and other query libraries) to clarify the missing features of PRQL
+and to highlight the latest features of DuckDB.
 
-At the same time, new features are being added to DuckDB's SQL on a constant basis,
-and the syntax is becoming more friendly.
+* [PRQL](https://prql-lang.org/) is a modern language for data manipulation
+that compiles to SQL.
+Although the PRQL language is still evolving,
+it is easy to create complex SQL queries. 
+* [DuckDB](https://duckdb.org/) is a modern in-memory
+([very fast](https://duckdblabs.github.io/db-benchmark/)) SQL OLAP
+database management system.
+DuckDB is also evolving, with new features and improving syntax.
 
-So, I thought that by comparing PRQL and the latest DuckDB SQL (and other query libraries),
-we could clarify the missing features of the PRQL and highlight the latest features of DuckDB.
+This project uses CLI, JavaScript, Python, R, etc.
```

Comment:
I don't think I am using CLI or JS within this document.

I just wanted to say that the combination of PRQL and DuckDB can be used in many different places.
Could you change the wording to something else?

Reply:
Check the new wording of this paragraph

Reply:
Sorry, but the new version seems to have completely removed the intent of the original document.
My point was that the combination of DuckDB and PRQL can be used in a variety of environments.

Reply:
Oh dear. I am sorry to have missed the meaning. I will close this PR, and I will watch for updates to the project.